scripts/verify-readme.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const path = require('path');
+
+const appRoot = process.cwd();
+
+// identify if it's node or web
+const isNode = appRoot.includes('node');
+const isWeb = appRoot.includes('web');
```

Comment:
This logic will break if we move to a single `./packages/...` dir (option 3 of https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2223#issuecomment-2125925648).

An alternative option would be to read `auto-instrumentations-*/package.json`, and look at their "dependencies" to see if a given `pjson.name` is included.

Reply:
done 👍